js/controllers/todos_controller.js

```diff
@@ -1,4 +1,125 @@
-App.TodosController = Ember.ObjectController.extend({
+
+Abs.TasksController = Ember.ObjectController.extend({
```

Comment:
Use controller to support multi view if need it. 


     On Monday, September 28, 2015 4:40 PM, Ruan Naude <notifications@github.com> wrote:
   

 In js/controllers/todos_controller.js:> @@ -1,4 +1,125 @@
> -App.TodosController = Ember.ObjectController.extend({
> +
> +Abs.TasksController = Ember.ObjectController.extend({
Why did you decide to create a controller to use for each Task?—
Reply to this email directly or view it on GitHub.